app/src/forms/email/emailService.js

```diff
@@ -105,10 +83,28 @@ const buildEmailTemplate = async (formId, formSubmissionId, templateType) => {
     };
   }
 
-  return { configData, submission };
+  const contexts = [{
+    context: {
+      allFormSubmissionUrl: `${service._appUrl(referer)}/user/submissions?f=${configData.form.id}`,
+      confirmationNumber: submission.confirmationId,
+      form: configData.form,
+      messageLinkText: configData.messageLinkText,
+      messageLinkUrl: `${service._appUrl(referer)}/${userTypePath}?s=${submission.id}`,
+      revisionNotificationEmailContent: additionalProperties.revisionNotificationEmailContent,
+      title: configData.title
+    },
+    to: contextToVal
+  }];
+
+  return { configData, contexts };
```

Comment:
We can probably collapse this into a single object return instead to avoid needing to do a single use const in block scope:
```js
return { configData, [
  {
    context: {
      allFormSubmissionUrl: `${service._appUrl(referer)}/user/submissions?f=${configData.form.id}`,
      confirmationNumber: submission.confirmationId,
      form: configData.form,
      messageLinkText: configData.messageLinkText,
      messageLinkUrl: `${service._appUrl(referer)}/${userTypePath}?s=${submission.id}`,
      revisionNotificationEmailContent: additionalProperties.revisionNotificationEmailContent,
      title: configData.title
    },
    to: contextToVal 
  }
]};
  ```